jest/env.js

```diff
@@ -36,7 +38,7 @@ global.test = (transformName, testFileName, options, fakeOptions) => {
     }
 
     expect(
-        (transform({path, source}, {jscodeshift}, options || {}) || "").trim()
+        (transform({path, fullPath, source}, {jscodeshift}, options || {}) || "").trim()
```

Comment:
I don't want to muck with this API too much. Can you see if you can use [`fakeOptions`](https://github.com/nfl/react-codemod/blob/master/test/__tests__/resolve-relative-imports-test.js#L7-L8) instead?